tests/foreman/api/test_registration.py

```diff
@@ -262,3 +262,61 @@ def test_negative_capsule_without_registration_enabled(
         "Proxy lacks one of the following features: 'Registration', 'Templates'"
         in context.value.response.text
     )
+
+
+@pytest.mark.rhel_ver_match('[^6]')
+def test_positive_host_registration_with_non_admin_user(
+    module_org,
+    module_location,
+    module_activation_key,
+    module_target_sat,
+    rhel_contenthost,
+):
+    """Verify host registration with non admin user
+
+    :id: 02bdda6a-010d-4098-a7e0-e4b5e8416ce3
+
+    :steps:
+        1. Enable the required repositories on the satellite and get a host(RHEL7/RHEL8/RHEL9)
+        2. Create a non-admin user and assign "Register Hosts" role to it.
+        3. Create an activation key and assign CV and LCE to it.
+        4. Login Satellite with new user and generate curl command to register host with all setup options set to NO.
```

Comment:
We're not generating curl command after login via UI, it's API test :)

Reply:
Changes has been done.